README.md

```diff
@@ -20,3 +20,36 @@ And this is going to stderr
 ```
 
 And yeah, GitHub-style Markdown doesn't really allow for colours. Let's just say the last line in the above output is now red.
+
+## Building:
+
+First ensure you have the `meson` and `ninja` binaries installed, as well as the
+required C compiler (`gcc`) and related tools.
+
+Run `meson build` in the project root directory.
+Next change into the build directory with `cd build/`
+Finally run `ninja` in that directory.
```

Comment:
most people would probably do "ninja -C build" rather than "cd build && ninja", but it doesn't really matter I guess..

Reply:
Woops! I actually didn't know, it was my first time using meson/ninja :P